pyproject.toml

```diff
@@ -62,3 +62,5 @@ test = {features = ["test"], solve-group = "default"}
 python = "3.12.*"
 pip = ">=25.0.1,<26"
 ipykernel = "*"
+psycopg2-binary = ">=2.9.9,<3"
+pymysql = "*"
```

Comment:
pymysql works as a pypi dependency only as well but for postgres I needed the dependency here - otherwise it needs pg-conf on the machine to be able to compile the driver